README.md

```diff
@@ -125,7 +100,9 @@ PNxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx  +1209337XXXX  DAVID'S TWILIO CONTACT
 
 Many `list` commands will allow you to specify a `--properties` option to specify which columns you'd like to display. For example, to display only the Phone Number and SMS Url columns, you would pass `--properties "phoneNumber, smsUrl"`.
 
-Note that currently, the column names must match the JSON property names in the Twilio API.
+Note that the default list of properties varies by command and is subject to change with each release.
```

Comment:
Should we guide them on best practices so that us changing properties wont break existing scripts?

Reply:
_Nobody's even going to read this thing_ (:

Sure, I'll add some verbiage.